### PR TITLE
Migrate TrackingStationAmbientlight Mod from Kerbalstuff to Spacedock.

### PR DESCRIPTION
**Pull request by mhoram-kerbin**
Please review before accepting.

### CODE REVIEW

File: NetKAN/TrackingStationAmbientlight.netkan [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "spec_version" : 1,
     "identifier"   : "TrackingStationAmbientlight",
-    "$kref"        : "#/ckan/kerbalstuff/1136",
+    "$kref"        : "#/ckan/spacedock/895",
     "$vref" : "#/ckan/ksp-avc",
     "x_netkan_license_ok": true,
```
Comment (politas) [inline NetKAN/TrackingStationAmbientlight.netkan]: Please replace this line with 

    "licence" : "GPL-3.0"

**Comment by politas** [comment]
With the licence fixed, I believe that will work. I note in your .version file, you still have kerbalstuff as the download source. That won't affect CKAN, but may impact AVC users who don't use CKAN.

**Comment by mhoram-kerbin** [comment]
@politas thanks for the review and your comment. Corrected the issues you mentioned.

**Comment by politas** [comment]
Looks good to me now. Merging with many thanks!

**Comment by mhoram-kerbin** [comment]
Thanks, it works fine!

**Pull request merged**